R/taxonomic-coverage-element.R

```diff
@@ -40,17 +40,17 @@
 #' name taxonomic_coverage to append it to the dataset or project. Example of how 
```

Comment:
lines 29-33 contain the ITIS identifier, pls add to EMLaid::CVPIA_common_species so the warning goes away

Reply:
Fixed warning